test/tomo/backends/astra_setup_test.py

```diff
@@ -73,11 +73,11 @@ def _discrete_domain_anisotropic(ndim, interp):
     space : `DiscreteLp`
         Returns a `DiscreteLp` instance
     """
-    minpt = [-1] * ndim
-    maxpt = [1] * ndim
+    min_pt = [-1] * ndim
+    max_pt = [1] * ndim
     nsamples = np.arange(1, ndim + 1, 1) * 10  # 10, 20 [, 30]
```

Comment:
Omit the third argument, it's default. And here I think its readable, no need for the extra comment.